docs/virtualserver-and-virtualserverroute.md

```diff
@@ -84,13 +84,12 @@ The tls field defines TLS configuration for a VirtualServer. For example:
 ```yaml
 secret: cafe-secret
 redirect:
-  code: 302
```

Comment:
I know this is based on specification but @pleshakov why do we need to remove code and basedOn from here? Seems to me adding `enable` and keep those is a more complete example?

Reply:
@Rulox we have a complete example in `VirtualServer.TLS.Redirect` section. My thinking was that here we can show a minimal config for tls redirect.

Reply:
I see thanks for clarification